Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -54,24 +54,33 @@ In order to configure an external Ceph cluster with Rook, we need to extract som
 - `--cephfs-data-pool-name`: (optional) Provides the name of the CephFS data pool, used for creating CephFS StorageClass
 - `--rados-namespace`: (optional) Divides a pool into separate logical namespaces, used for creating RBD PVC in a RadosNamespaces
 - `--subvolume-group`: (optional) Provides the name of the subvolume group, used for creating CephFS PVC in a subvolumeGroup
+
+Multi-tenancy:
```

Comment:
How about moving this section for multi-tenancy down after the "sample run" and before the "upgrades" section? Then the parameter `--restricted-auth-permission` can stay with the other parameters without a paragraph between the parameters.
```suggestion
#### Multi-tenancy
```